### PR TITLE
wide tables

### PR DESCRIPTION
**Pull request by fabothch**
Make text in tables break and continue in new line, rather than continue and force the reader to scroll down to the end of the table and move the horizontal handle.

### CODE REVIEW

**Comment by awvwgk** [comment]
Preview available here: https://xtb-docs.readthedocs.io/en/tables/contents.html

File: source/conf.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
 #import sys
 #sys.path.insert(0, os.path.abspath('.'))
 
-
+def setup(app):
+    app.add_css_file('custom.css')
```
Comment (awvwgk) [inline source/conf.py:18]: Use:
```python
html_css_files = [
    "custom.css",
]
```
instead

Comment (fabothch) [reply source/conf.py]: changed

**Review by awvwgk — APPROVED** [review]
Thanks for sharing, looks good to me.

**Pull request merged**